ipfx/ephys_data_set.py

```diff
@@ -149,6 +150,9 @@ def sweep(self, sweep_number):
         else:
             raise Exception("Unable to determine clamp mode for sweep " + sweep_number)
 
+        v *= 1.0e3    # convert units V->mV
```

Comment:
If you are getting `sweep_data` from the nwb, it would be a good idea to check that the units of the data are actually in `V` and `A`. At least based on the description of that units field, it could potentially be in `mv` or `pA` already, especially if the data comes from a different source?

Reply:
Good point, I have added a validation function to check that both stimulus and response use SI units.